Packs/Base/Scripts/CommonServerPython/CommonServerPython.yml

```diff
@@ -13,7 +13,6 @@ system: true
 scripttarget: 0
 dependson: {}
 timeout: 0s
-alt_dockerimages: # used for unit testing
-  - demisto/python3:3.7.3.286
+dockerimage: demisto/python3:3.8.3.8494   # used for unit testing
```

Comment:
is the `alt_dockerimages` still needed? or can we change that? @glicht

Reply:
Yes it is needed.

Reply:
Also don't add the docker image tag as then we are NOT using the default image.